### PR TITLE
feat(EMS-2287 2347 2363): No PDF - Single/multiple policy - currency enhancements

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR updates single/multiple contract currency support to include `JPY` and improves the UX/UI design. This is for both a no PDF application and the quote tool.

## Resolution :heavy_check_mark:
- Add `JPY` to `CURRENCIES` fixtures and `SUPPORTED_CURRENCIES` constant. Change `CURRENCIES` into an object structure.
- Create new cypress command, `changeAnswerRadioField`, similar to `changeAnswerSelectField`.
- Create shared "radios" cypress selector function.
- Update policy currency legend/label and error message copy.
- Update various E2E tests to submit/check a policy's currency as a radio option, instead of a select option.
- Simplify "map currencies" logic in the single and multiple contract policy controllers.
- Replace `mapCurrencies` function with `mapCurrenciesAsRadioOptions` and `mapCurrenciesAsSelectOptions`, since they require different mappings.
- Update application single and multiple contract policy nunjucks template to render currency radios with a new component, `currency-radio-inputs.njk`.

## Miscellaneous :heavy_plus_sign:
- Extract "get supported countries" function into its own function/directory.
- Create new helper function to generate a country/currency name with its ISO code, `nameAndIsoCodeText`. Returns eg. "the country (isoCode)".
- Add some missing E2E `POLICY` constants.
- Remove unrequired `SUPPORTED_CURRENCIES` E2E constant.
- Fix some typos.
- Remove some unnecessary nunjuck imports.

### CODE REVIEW

File: e2e-tests/commands/shared-commands/form/change-answer-radio-field.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { radios, submitButton } from '../../../pages/shared';
+/**
+ * changes radio input/option
+ * @param {Object} fieldVariables
+ * @param {Function} input cypress input selector
```
Comment (Zainzzkk) [inline e2e-tests/commands/shared-commands/form/change-answer-radio-field.js:5]: only 1 param being passed.  SHould this be returns?

File: src/ui/server/helpers/mappings/map-currencies/as-select-options/index.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * mapCurrenciesAsSelectOptions
+ * Map all currencies into the required structure for GOV select component.
+ * @param {Array} Array of currency objects
+ * @param {String} Selected currency
+ * @returns {Array} Array of mapped currencies
+ */
+const mapCurrenciesAsSelectOptions = (currencies: Array<Currency>, selectedValue?: string) => {
+  const supportedCurrencies = getSupportedCurrencies(currencies);
+
+  const mapped = supportedCurrencies.map(({ name, isoCode }) => mapSelectOption(name, isoCode, true, selectedValue));
```
Comment (Zainzzkk) [inline src/ui/server/helpers/mappings/map-currencies/as-select-options/index.ts:16]: should we name a variable as true so we know exactly what is being set as true without going to the function?

File: e2e-tests/commands/shared-commands/form/change-answer-radio-field.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/commands/shared-commands/form/change-answer-radio-field.js:5]: Typo, removed

File: src/ui/server/helpers/mappings/map-currencies/as-select-options/index.ts (hunk 2 above)
Comment (ttbarnes) [reply src/ui/server/helpers/mappings/map-currencies/as-select-options/index.ts:16]: This has been around for ages 😄  but good shout, have updated

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**